src/wallet/wallet.h

```diff
@@ -1200,4 +1200,7 @@ class WalletRescanReserver
 // be IsAllFromMe).
 int64_t CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *wallet, bool use_max_sig = false);
 int64_t CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *wallet, const std::vector<CTxOut>& txouts, bool use_max_sig = false);
+
+// Calculate the size of a single input of type P2SH-P2WPKH
+size_t CalculateNestedKeyhashInputSize(bool use_max_sig = false);
```

Comment:
nit: If this is exposed here for testing, declaring it `extern` in the test file instead will reduce the internal surface area

Reply:
As of now, it can be removed as the method is now in the wallet_tests file.